src/services/mongoose/accounts.ts

```diff
@@ -166,7 +166,10 @@ export const AccountsRepository = (): IAccountsRepository => {
     kratosUserId: UserId,
   ): Promise<Account | RepositoryError> => {
     try {
-      const result = await Account.findOne({ kratosUserId })
+      const result = await Account.findOne(
+        { kratosUserId: { $eq: kratosUserId } },
+        { lastIPs: 0 },
```

Comment:
not sure about this... why findByUserId should depends on lastIPs?

Reply:
it does not. ip is fetched from another service (`accounts-ip`). the fact it's not excluded is a just a miss. it's been excluded from other queries in this file.